spec/ruby/core/range/count_spec.rb

```diff
@@ -0,0 +1,20 @@
+require_relative '../../spec_helper'
+
+describe "Range#count" do
+  ruby_version_is "2.6" do
+    it "returns Infinity for endless ranges without arguments or blocks" do
+      inf = Float::INFINITY
+      ("a"...).count.should == inf
+      (7..).count.should == inf
+    end
+  end
+
+  ruby_version_is "2.7" do
+    it "returns Infinity for beginless ranges without arguments or blocks" do
+      inf = Float::INFINITY
+      (..."p").count.should == inf
+      (...nil).count.should == inf
+      (..10.0).count.should == inf
```

Comment:
Should use `eval` so that it will be able to parse on older Ruby versions. In other places too, both for beginless and endless ranges.

Reply:
The GH Actions check should catch that, but it unfortunately fails since yesterday. Should be fixed soon.

Reply:
Should be fixed now, any new commit/push should trigger the usual actions.

Reply:
That is so useful know. GH actions is also so useful; there were so many bugs that I snuck through while it was off. Sorry!